### PR TITLE
Add Gruvbox-Dark and Gruvbox-Light themes

### PR DESCRIPTION
**Pull request by MarshDeer**

### CODE REVIEW

File: src/components/shared/setDefaultSettings.ts [hunk 1]
```diff
@@ -1579,6 +1579,328 @@ const setDefaultSettings = (force: boolean) => {
...
+        tooltip: {
+          color: '#EBDBB2',
+          background: '#1D2021'
+        }
+      },
+      other: {
+        button: {
+          borderRadius: '15px'
+        },
+        coverArtBorderRadius': '5px',
```
Comment (jeffvli) [inline src/components/shared/setDefaultSettings.ts:1693]: There's an extra `'` on `coverArtBorderRadius'`, needs to be removed

File: src/components/shared/setDefaultSettings.ts [hunk 2]
```diff
@@ -1579,6 +1579,328 @@ const setDefaultSettings = (force: boolean) => {
...
+        coverArtFilter: none',
```
Comment (jeffvli) [inline src/components/shared/setDefaultSettings.ts:1694]: Similarly, the closing `'` is missing for `none'`

File: src/components/shared/setDefaultSettings.ts [hunk 3]
```diff
@@ -1579,6 +1579,328 @@ const setDefaultSettings = (force: boolean) => {
...
+          color: '#3C3836',
+          background: '#EBDBB2'
+        }
+      },
+      other: {
+        button: {
+          borderRadius: '15px'
+        },
+        coverArtBorderRadius': '3px',
+        coverArtFilter: none',
```
Comment (jeffvli) [inline src/components/shared/setDefaultSettings.ts:1855]: Same thing here.

**Comment by jeffvli** [comment]
Looks pretty good! A couple suggestions from me:

1. Convert the color to a rgba value and add opacity for the `colors.table.selectedRow` color value since solid colors can make it harder to read what's currently selected.
image
image

2. I think the `colors.button.default.colorHover` is broken (from my code side) so readability is difficult when hovering buttons on Gruvbox Light. I'm currently doing a full application rewrite so I probably won't be fixing it for the time being. I'd suggest choosing a different color and background hover color for the buttons so it's readable.
image

**Comment by MarshDeer** [comment]
Did the requested fixes and followed the workflow's recommendations re: adding `,` at the end of some lines. 
Themes should hopefully be properly functional now!

**Comment by jeffvli** [comment]
Thanks!

**Pull request merged**